##
# LabeledFaces.msg - contains a label and the IDs of all faces with that label
##
Header header
string label
uint32[] ids
